src/tools/dev/scripts/bv_support/helper_funcs.sh

```diff
@@ -856,24 +856,28 @@ function check_files
 
 
 # *************************************************************************** #
-#                          extract_parallel_ldflags                           #
+#                          process_parallel_ldflags                           #
 # --------------------------------------------------------------------------- #
-# VisIt's cmake config wants lib names stripped of "-l"                       #
-# If PAR_LIBS is used to pass parallel LDFLAGS we need to separate the libs   #
-# from the linker flags and strip the "-l" prefixes.                          #
-# This function accomplishes this and creates two new  variables:             #
-#   PAR_LINKER_FLAGS & PAR_LIBRARY_NAMES                                      #
+# This routine processes the PAR_LIBS variable into three other variables.    #
+#   PAR_LINKER_FLAGS : Any linker flags that aren't libraries (don't start    #
+#                      with "-l".                                             #
+#   PAR_LIBRARY_NAMES: The library names with the "-l" stripped out.          #
+#   PAR_LIBRARIES    : The library names with the "-l".                       #
```

Comment:
Are you saying rename PAR_LINKER_FLAGS to PAR_LIBRARY_LINKER_FLAGS? I could certainly make that change here and in the files that reference it. I didn't actually introduce that name, it was already there. I added PAR_LIBRARIES, which may not be a great name, but that's what I came up with. Perhaps I could rename PAR_LIBRARIES to PAR_LIBRARY_NAMES_WITH_DASH_L_PREFIX?

Reply:
Am suggesting `PAR_LIBRARIES` --> `PAR_LIBRARY_LINKER_FLAGS` because it represents the fusion of the two other variable's functionality. Your call though.

Reply:
Okay, I misunderstood. Just so we are clear, PAR_LIBS gets split into PAR_LINKER_FLAGS and PAR_LIBRARIES. PAR_LIBRARY_NAMES is PAR_LIBRARIES with the leading "-l" stripped from each library name. I think your change makes sense now. I'll rename it.